forge-gui/res/cardsfolder/upcoming/staff_of_the_storyteller.txt

```diff
@@ -0,0 +1,10 @@
+Name:Staff of the Storyteller
+ManaCost:1 W
+Types:Artifact
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When CARDNAME enters the battlefield, create a 1/1 white Spirit creature token with flying.
+SVar:TrigToken:DB$ Token | TokenAmount$ 1 | TokenScript$ w_1_1_spirit_flying | TokenOwner$ You
+T:Mode$ TokenCreated | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigPutCounter | TriggerDescription$ Whenever you create one or more creature tokens, put a story counter on CARDNAME.
```

Comment:
needs new Trigger because of "one or more"

Reply:
@tool4ever is that a work for me? i thought we already had something like that?

Reply:
Well on _Akim, the Soaring Wind_ it was cheatable because he also had "for the first time each turn"

I'll see if I can do it tonight, shouldn't be fairly straight forward?

Reply:
yeah, otherwise i will look into it later